### PR TITLE
Fix typo 

### PR DESCRIPTION
**Pull request by FarukhS52**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ of [Conda](https://docs.conda.io/en/latest/):
 
 > Ruff is ~150-200x faster than flake8 on my machine, scanning the whole repo takes ~0.2s instead of
 > ~20s. This is an enormous quality of life improvement for local dev. It's fast enough that I added
-> it as an actual commit hook, which is terrific.
+> it is an actual commit hook, which is terrific.
```
Comment (zanieb) [inline README.md:83]: This one was correct previously

**Review by zanieb — COMMENTED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (FarukhS52) [reply README.md:83]: Ohhh my bad

Comment (FarukhS52) [reply README.md:83]: If i change it back as earlier then it's all good??

**Review by zanieb — APPROVED** [review]

**Pull request merged**